specificatie/genereervariant/openapi.json

```diff
@@ -342,6 +343,22 @@
           }
         } ]
       },
+      "ZoekMetAdresseerbaarObjectIdentificatieResponse" : {
+        "allOf" : [ {
+          "$ref" : "#/components/schemas/PersonenQueryResponse"
+        }, {
+          "type" : "object",
+          "properties" : {
+            "personen" : {
+              "type" : "array",
+              "description" : "* **geslacht** - wordt gevuld met waarden voor 'Geslacht' in 'tabelwaarden.csv'.\n",
```

Comment:
Deze description lijkt me nog niet helemaal correct :-)

Reply:
gaan we aanpassen :-)